src/main/java/iroha/validation/config/ValidationServiceContext.java

```diff
@@ -0,0 +1,50 @@
+package iroha.validation.config;
+
+import iroha.validation.transactions.signatory.TransactionSigner;
+import iroha.validation.transactions.storage.TransactionProvider;
+import iroha.validation.validators.Validator;
+import java.security.KeyPair;
+import java.util.Collection;
+import java.util.Objects;
+
+public class ValidationServiceContext {
+
+  private final Collection<Validator> validators;
+  private final TransactionProvider transactionProvider;
+  private final TransactionSigner transactionSigner;
+  private final KeyPair keyPair;
+
+  public ValidationServiceContext(
+      Collection<Validator> validators,
+      TransactionProvider transactionProvider,
+      TransactionSigner transactionSigner,
+      KeyPair keyPair) {
+    Objects.requireNonNull(validators, "Validators collection must not be null");
+    // TODO rework isEmpty check during springify task
+    if(validators.isEmpty()) throw new RuntimeException("Validators collection must not be empty");
```

Comment:
I think it's better to throw `IllegalArgumentException`.

Reply:
Done